.github/workflows/ocl-sub-module.yml

```diff
@@ -0,0 +1,28 @@
+name: OCL Subscription Module
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Comment:
Why not renaming the workflow as ocl-subscription.yml

Reply:
Sounds better,  i will do that.